src/Core/Content/Product/SalesChannel/Search/ProductSearchRoute.php

```diff
@@ -101,6 +102,11 @@ public function load(
         SalesChannelContext $context,
         ?Criteria $criteria = null
     ): ProductSearchRouteResponse {
+
+        if (!Utils::versionLowerThan('6.4.0.0')) {
```

Comment:
:thinking: What do you think about a utils function called something like `Utils::versionAboveOrEqual`, which represents (in this example) all versions starting from `6.4.0.0`.
It's not like this is some complex statement, but it would make it a bit more clear.

Reply:
Agree, having a separate method for version greater or equal makes sense.